<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IncluirLivro</class>
 <widget class="QDialog" name="IncluirLivro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>206</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Incluir Livro</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>116</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>INCLUIR NOVO LIVRO</string>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>166</width>
     <height>24</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNome">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>39</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPreco">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pre√ßo: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>120</y>
     <width>36</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Autor: </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAutor">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>IncluirLivro</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>IncluirLivro</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
